cmd/mysterium_client/cli/command.go

```diff
@@ -202,11 +202,19 @@ func (c *Command) status() {
 	status, err := c.tequilapi.Status()
 	if err != nil {
 		warn(err)
-		return
+	} else {
+		info("Status:", status.Status)
+		info("SID:", status.SessionID)
 	}
 
-	info("Status:", status.Status)
-	info("SID:", status.SessionId)
+	statistics, err := c.tequilapi.ConnectionStatistics()
```

Comment:
It would be nice if statistics are printed only if status == connected, otherwise - why to trash screen with not relevant information?

Reply:
Updated